res/css/views/settings/tabs/user/_HelpUserSettingsTab.pcss

```diff
@@ -28,4 +33,9 @@ limitations under the License.
             margin-bottom: $spacing-16;
         }
     }
+
+    div .mx_CopyableText {
+        overflow-x: scroll;
+        overflow-y: scroll;
```

Comment:
why do we set `overflow-x` and `overflow-y` separately, rather than just setting `overflow`?

Reply:
This I did was for experimenting earlier I was applying only `overflow-x` that's why I additionally added `overflow-y` for checking in dev tools. I forgot to change it to `overflow`.